sync-version/action.yaml

```diff
@@ -52,6 +49,7 @@ runs:
            exit 1
         fi
         echo "yaml_path=${yaml_path}" >> "$GITHUB_OUTPUT"
+        echo "snap-name=$(yq -r '.name' "$yaml_path")" >> "$GITHUB_OUTPUT"
         echo "prev_version=$(yq -r '.version' "$yaml_path")" >> "$GITHUB_OUTPUT"
```

Comment:
Can you stick to a convention with naming - as in all `-` or all `_` ? FWIW I think `-` is easier for people to type :)